felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -3809,6 +3776,11 @@ func (m *bpfEndpointManager) loadPolicyProgram(
 		ipSetIDAlloc = m.v6.ipSetIDAlloc
 	}
 
+	// Flow logs functionality is enabled.
+	if m.lookupsCache != nil {
```

Comment:
nit, perhaps wrap the test in a method so the code reads better, rather than needing a comment?
```suggestion
	if m.FlowLogsEnabled() {
```

Reply:
make sense. Will fix it.